stdlib/publickey.go

```diff
@@ -333,8 +335,10 @@ func NativePublicKeyVerifyPoPFunction(
 	) interpreter.Value {
 		signatureValue := interpreter.AssertValueOfType[*interpreter.ArrayValue](args[0])
 
+		publicKeyValue := constantPublicKeyValue
```

Comment:
Shouldn't we **not** cache the value? 🤔

Reply:
Sorry, I don't quite understand the question/concern. `constantPublicKeyValue` is passed by the interpreter, but is `nil` in the VM. So at call-time we use it if it was passed (interpreter), or use the current receiver.